uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -558,8 +595,16 @@ sub updateMRIUploadTable  {
         my $where = "WHERE UploadID=?";
         my $query = "UPDATE mri_upload SET $field=?";
         $query = $query . $where;
+        (my $query = <<QUERY) =~ s/\n/ /gm;
```

Comment:
@cmadjar,

you do not need the "my" before $query here (defined 2 lines earlier)